web/src/user_guide/Page.svelte

```diff
@@ -0,0 +1,518 @@
+<script lang="ts">
+  import "@picocss/pico/css/pico.conditional.jade.min.css";
+  import Toc from "svelte-toc";
+  import About from "../About.svelte";
+  import image01 from "./image01.webp?url";
```

Comment:
Vite is kind of annoying here, I don't know how to avoid this boilerplate. When I've used the static assets directory in the past, I've hit every combination of headaches making it work on GH Pages vs locally vs on a domain without a leading base path.